File: raejercicio10.cpp
#include <iostream>

using namespace std;

int main() {
    int numeros[10]; 

    cout << "Ingresa 10 números:" << endl;
    for (int i = 0; i < 10; ++i) {
        cout << "Número " << (i + 1) << ": ";
        cin >> numeros[i]; 
    }

    cout << "Números pares en el array:" << endl;
    for (int i = 0; i < 10; ++i) {
        if (numeros[i] % 2 == 0) { 
            cout << numeros[i] << " "; 
        }
    }
    cout << endl; 

    return 0;
}

File: raejercicio10.js
let numeros = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
let numerosImpares = [];

for (let i = 0; i < numeros.length; i++) {

    if (numeros[i] % 2 !== 0) {
        numerosImpares.push(numeros[i]);
    }
}

console.log("Números impares:", numerosImpares);